blog/convert_kis_svg_to_png.py

```
@@ -0,0 +1,108 @@
+
+    async with async_playwright() as p:
+        browser = await p.chromium.launch()
+        page = await browser.new_page()
+
+        # HTML 설정
+        await page.set_content(html_content)
+
+        # SVG 요소 찾기
+        svg_element = await page.query_selector('svg')
+        if not svg_element:
+            raise ValueError(f"SVG element not found in {svg_path}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)